bitcoin_executor/sources/interpreter.move

```diff
@@ -308,21 +308,35 @@ const SIG_VERSION_WITNESS_V0: u8 = 1; //SEGWIT
 
 /// Hash types
 const SHA256: u8 = 1;
+const SUCCESS: u64 = 0;
 
 // ============= Errors ================================
 #[error]
 const EEqualVerify: vector<u8> = b"SCRIPT_ERR_EQUALVERIFY";
 #[error]
-const EInvalidStackOperation: vector<u8> = b"Invalid stack operation";
-#[error]
-const EMissingTxCtx: vector<u8> = b"Missing transaction context";
-#[error]
 const EUnsupportedSigVersionForChecksig: vector<u8> =
     b"Unsupported signature version for op_checksig";
 #[error]
-const EInvalidOpcode: vector<u8> = b"Invalid Opcode";
-#[error]
-const EInternalBitcoinCoreOpcode: vector<u8> = b"Invalid Opcode: Bitcoin core internal";
+const EInvalidPKHLength: vector<u8> = b"Invalid PHK: PHK should have length is 20";
+
+const EPopStackEmpty: u64 = 1;
+const ETopStackEmpty: u64 = 2;
+const EMissingTxCtx: u64 = 3;
+const EInvalidOpcode: u64 = 4;
+const EInternalBitcoinCoreOpcode: u64 = 5;
```

Comment:
why mixing legacy errors with `#[error]`?

Reply:
We return a legacy error when the BTC script failed. This is a u64 type, but the new error is a string type. 
This is why we have 2 error types here. One for return to user other to abort